site/_data/exampleapps.yaml

```diff
@@ -267,10 +267,6 @@
   name: FusionDesk (React SDK)
   description: React/express app illustrating the use of the React SDK and Hosted OAuth Service Provider Endpoints
   language: react
-- url: https://github.com/FusionAuth/fusionauth-example-express-api
```

Comment:
I see we are removing the old one, shouldn't we add this one?

Reply:
added the new one